packages/server/src/environment.ts

```diff
@@ -72,9 +72,9 @@ const environment = {
   HTTP_MB_LIMIT: process.env.HTTP_MB_LIMIT,
   FORKED_PROCESS_NAME: process.env.FORKED_PROCESS_NAME || "main",
   JS_PER_INVOCATION_TIMEOUT_MS:
-    parseIntSafe(process.env.JS_PER_INVOCATION_TIMEOUT_MS) || 1000,
+    parseIntSafe(process.env.JS_PER_EXECUTION_TIME_LIMIT_MS) || 1000,
   JS_PER_REQUEST_TIMEOUT_MS: parseIntSafe(
-    process.env.JS_PER_REQUEST_TIMEOUT_MS
+    process.env.JS_PER_REQUEST_TIME_LIMIT_MS
```

Comment:
Not sure I understand why these are changing. Did we rename them with the release of the isolated-vm changes?

Reply:
Yes, that was caused by a wrong renaming. Changing it back to what it was